tests/system/action/list_of_speakers/test_re_add_last.py

```diff
@@ -112,3 +143,18 @@ def test_last_speaker_also_in_waiting_list(self) -> None:
         self.assertTrue(
             "User 42 is already on the list of speakers." in response.json["message"]
         )
+
+    def test_re_add_last_no_permissions(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "list_of_speakers.re_add_last",
```

Comment:
You are mixing the string version and the permissions from the enums. I would prefer using the enums.

Reply:
`"list_of_speakers.re_add_last"` is the action, not a permission. We **always** use the action string.

Reply:
Right. Ok!